src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -98,7 +100,7 @@ public class ExtensionsRunner {
     private ExtensionsIndicesModuleNameRequestHandler extensionsIndicesModuleNameRequestHandler =
         new ExtensionsIndicesModuleNameRequestHandler();
     private ExtensionsRestRequestHandler extensionsRestRequestHandler = new ExtensionsRestRequestHandler();
-    private NettyTransport nettyTransport = new NettyTransport();
+    private NettyTransport nettyTransport = new NettyTransport(this);
```

Comment:
I really like this pattern for a lot of the refactorings which need to know what the ExtensionsRunner object is.